scripts/start.sh

```diff
@@ -9,7 +9,7 @@ ETH1_CHAIN_ID=1
 TARGET_GAS_LIMIT=9000000
 CHAIN_ID=10
 ETH1_CTC_DEPLOYMENT_HEIGHT=11650235
-ETH1_L1_CROSS_DOMAIN_MESSENGER_ADDRESS=0x0AEBf5161A9b57349747D078c6763a0B1d67D888
+ETH1_L1_CROSS_DOMAIN_MESSENGER_ADDRESS=0xfBE93ba0a2Df92A8e8D40cE00acCF9248a6Fc812
```

Comment:
This fixes the default value of the xdomain messenger, see
 ` "Proxy__OVM_L1CrossDomainMessenger": "0xfBE93ba0a2Df92A8e8D40cE00acCF9248a6Fc812",`
https://github.com/ethereum-optimism/regenesis/tree/master/mainnet